src/BenchmarkDotNet/Toolchains/MonoWasm/WasmAppBuilder.cs

```diff
@@ -0,0 +1,201 @@
+// -*- indent-tabs-mode: nil -*-
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Reflection;
+using BenchmarkDotNet.Toolchains.MonoWasm;
+
+public class WasmAppBuilder
+{
+    // FIXME: Document
+
+    private readonly WasmSettings WasmSettings;
+
+    private readonly string TargetFrameworkMoniker;
+
+  
+
+    private Dictionary<string, Assembly> _assemblies;
+    private AssemblyResolver _resolver;
+
+
```

Comment:
remove the unnecessary newlines:

```suggestion

    private Dictionary<string, Assembly> _assemblies;
    private AssemblyResolver _resolver;

```